majors_and_certificates/certificates/AFS.yaml

```diff
@@ -0,0 +1,44 @@
+---
+type: Certificate
+name: African Studies
+code: AFS
+degree:
+urls:
+- https://afs.princeton.edu/undergraduate-opportunities/certificate-african-studies
+description: The program offers classes in Africa’s political, economic, and social history; 
+built environments and urban geographies; ecology, genetic diversity, and epidemiological concerns. 
```

Comment:
Make sure to indent these lines

Reply:
These lines are part of the quote from the previous line, so they should be indented to show nesting.

Reply:
Even better, use one of YAML's block string formats
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines